static/styles/zulip.css

```diff
@@ -1585,7 +1605,8 @@ div.focused_table {
 }
 
 .messagebox-content {
-    padding: 4px 115px 1px 10px;
+    padding: 2px 115px 1px 10px;
+    margin-left: 2px; /* stream color bar */
 }
```

Comment:
This misaligns the left side of the blue focus box.

Reply:
I honestly don't hate the way that looks, but yeah that wasn't on purpose. I'll wait a bit, ask how people feel about having hover functionality in general and also how this looks, and then probably do some finagling to get a new div setup for hover.

Reply:
thanks!